### PR TITLE
 Don't kill master server if replica crash was expected

### PR DESCRIPTION
**Pull request by sergw**

### CODE REVIEW

File: lib/tarantool_server.py [hunk 1]
```diff
@@ -106,7 +106,14 @@ def killall_servers(self, server, ts, crash_occured):
 
         # check that all servers stopped correctly
         for server in check_list:
-            crash_occured = crash_occured or server.process.returncode not in (None, 0, -signal.SIGKILL, -signal.SIGTERM)
+            bad_returncode = server.process.returncode not in (None,
+                                                               0,
+                                                               -signal.SIGKILL,
+                                                               -signal.SIGTERM)
+            # if replica crashed but this crash is exptected
```
Comment (Totktonada) [inline lib/tarantool_server.py]: exptected → expected

Comment (Totktonada) [reply lib/tarantool_server.py]: From test-run point there are default and non-default servers. We can write it as 'non-default server (say, replica)'.

File: lib/tarantool_server.py [hunk 2]
```diff
@@ -106,7 +106,14 @@ def killall_servers(self, server, ts, crash_occured):
...
+            # don't kill the master server and crash detectors
```
Comment (Totktonada) [inline lib/tarantool_server.py]: master server → default server (say, replication master)

**Comment by sergepetrenko** [comment]
@Totktonada The fix works fine.

**Pull request merged**